libfastpm/cosmology.c

```diff
@@ -273,33 +272,44 @@ static ode_soln growth_ode_solve(double a, FastPMCosmology * c)
     gsl_odeiv2_driver * drive 
         = gsl_odeiv2_driver_alloc_standard_new(&F,
                                                gsl_odeiv2_step_rkf45, 
-                                               1e-5, 
+                                               1e-6,
```

Comment:
Is this the initial time? What is the effect going from -5 to -6?

Reply:
No, this is the initial step size for the ODE solver. The initial time is set separately in the code as aini. I'm still testing this to find the best choice.